js/components/textTrackPanel.js

```diff
@@ -127,8 +129,8 @@ var TextTrackPanel = React.createClass({
                   this.props.closedCaptionOptions.textEnhancement
                 )
               }
-              >
-              {this.props.cueText}
+             >
+             <span dangerouslySetInnerHTML={Utils.createMarkup(this.props.cueText)}></span>
```

Comment:
https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml

`dangerouslySetInnerHTML` is React's replacement for using `innerHTML` in the browser DOM. In general, setting HTML from code is risky because it's easy to inadvertently expose your users to a [cross-site scripting (XSS)](https://en.wikipedia.org/wiki/Cross-site_scripting) attack. So, you can set HTML directly from React, but you have to type out `dangerouslySetInnerHTML` and pass an object with a` __html` key, to remind yourself that it's dangerous.